apps/web/templates/download-link.ts

```diff
@@ -30,7 +42,13 @@ html
         p   #{name}
         p 
             |   To access all of your content, 
-            |   #[a(href=\`\${loginLink}\`) log in] here. 
+            |   #[a(href=\`\${loginLink}\`) log in] here.
```

Comment:
No conditional check here.

Reply:
Why we nee conditional check here.

Reply:
So that when the `Hide CourseLit branding` checkbox is clicked, this badge is not visible in the emails

Reply:
Added condition